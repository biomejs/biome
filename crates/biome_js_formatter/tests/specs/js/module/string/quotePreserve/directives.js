'use preferred quote';
"use preferred quote";
"keep quotes and escapes \" ";
'keep quotes and escapes \' ';