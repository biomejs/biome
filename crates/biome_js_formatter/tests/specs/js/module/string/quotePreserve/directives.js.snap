---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: js/module/string/quotePreserve/directives.js
---
# Input

```js
'use preferred quote';
"use preferred quote";
"keep quotes and escpaes \" ";
'keep quotes and escpaes \' ';
```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Quote style: Double Quotes
JSX quote style: Double Quotes
Quote properties: As needed
Trailing comma: All
Semicolons: Always
Arrow parentheses: Always
Bracket spacing: true
Bracket same line: false
Attribute Position: Auto
-----

```js
"use preferred quote";
"use preferred quote";
"keep quotes and escpaes \" ";
'keep quotes and escpaes \' ';
```

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Quote style: Double Quotes
JSX quote style: Double Quotes
Quote properties: Preserve
Trailing comma: All
Semicolons: Always
Arrow parentheses: Always
Bracket spacing: true
Bracket same line: false
Attribute Position: Auto
-----

```js
"use preferred quote";
"use preferred quote";
"keep quotes and escpaes \" ";
'keep quotes and escpaes \' ';
```
